seshdash/models.py

```diff
@@ -37,6 +37,23 @@ class Meta:
         verbose_name = "RMC API Account"
 
 
+class Sesh_User(models.Model):
+    #TODO each user will have his her own settings / alarms this needs
+    #to be added
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    department = models.CharField(max_length=100)
+    phone_number =  models.CharField(max_length=12)
```

Comment:
The phone number should be possible that it's blank or empty. Subsequently in your code you should check if the user has a phone number